### ISSUE TITLE
Accessibility: Content not inlcuded in landmarks

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#289]
Hello, I'm also receiving a flag on the following highlighted text saying that the content isn't included in the HTML5 or WAI-ARIA landmarks. I think this also has something to do with the new e-mail form? Thanks!

![capture2](https://user-images.githubusercontent.com/15264315/54614282-88799380-4a32-11e9-8fa5-15581026907c.JPG)

**Comment by d4mation** [comment]
I've pushed a change to Staging to wrap this within a Landmark just after the pop-over is created. This should be set now 👍 Can you confirm for me?

**Comment by banksashleym** [comment]
This looks good too!

**Comment by d4mation** [comment]
This change is now Live 👍
